pages/touring-series/[slug].vue

```diff
@@ -76,34 +129,150 @@ useHead({
     }
   ]
 })
-
-// const pageClasses = computed(() => {
-//   return ['page', 'page-detail', 'page-detail--paleblue', 'page-touring-series-detail', 'page-bottom-spacer']
-// })
 </script>
 
 <template>
   <main
     id="main"
-    :class="pageClasses"
+    class="page page-detail page-detail--paleblue page-touring-series-detail"
   >
-    <SectionWrapper>
-      <div class="one-column">
-        <h3>
-          <pre>PAGE:<br/> {{ page }}</pre>
-        </h3>
-        <hr />
-        <hr />
-        <h3>
-          <pre>SERIES:<br/> {{ series }}</pre>
-        </h3>
+    <div class="one-column">
+      <NavBreadcrumb
+        class="breadcrumb"
+        :title="page?.title"
+      />
+
+      <ResponsiveImage
+        v-if="parsedImage.length === 1"
+        :media="parsedImage[0].image[0]"
+        :aspect-ratio="43.103"
+      >
+        <template
+          v-if="parsedImage[0]?.creditText"
+          #credit
+        >
+          {{ parsedImage[0]?.creditText }}
+        </template>
+      </ResponsiveImage>
+      <div
+        v-else
+        class="lightbox-container"
+      >
+        <FlexibleMediaGalleryNewLightbox
+          v-if="parsedCarouselData && parsedCarouselData.length > 0"
+          :items="parsedCarouselData"
+          :inline="true"
+        >
+          <template #default="slotProps">
+            <BlockTag :label="parsedCarouselData[slotProps.selectionIndex]?.creditText" />
+          </template>
+        </FlexibleMediaGalleryNewLightbox>
       </div>
+    </div>
+
+    <TwoColLayoutWStickySideBar>
+      <template #primaryTop>
+        <CardMeta
+          category="Series"
+          :title="page?.title"
+          :introduction="page?.ftvaEventIntroduction"
+        />
+      </template>
+
+      <template #primaryMid>
+        <RichText
+          v-if="page?.richText"
+          :rich-text-content="page?.richText"
+        />
+
+        <SectionHeader
+        :level="2"
+        class="section-header"
+        data-test="section-header"
+        >
+          Tour Dates
+        </SectionHeader>
+        <RichText
+          v-if="page?.richText"
+          class="tour-dates"
+          :rich-text-content="page?.richTextDefaultWithTable"
+        />
+      </template>
+
+      <!-- Sidebar -->
+      <template #sidebarTop>
+        <BlockEventDetail
+          data-test="event-details"
+          :start-date="page?.startDate"
+          :end-date="page?.endDate"
+          :ongoing="page?.ongoing"
+          :locations="page?.location"
```

Comment:
I do not see any data coming in from gql for locations and ongoing props here https://github.com/UCLALibrary/ftva-website-nuxt/blob/2a3899052cffae2bed3e5adc1e36fd264851a68e/gql/queries/FTVATouringSeriesDetail.gql#L4

Reply:
<img width="427" height="157" alt="Screenshot 2025-10-21 at 4 35 19 PM" src="https://github.com/user-attachments/assets/1950aff1-c971-45e0-864d-deb26df9f39e" />